runtime/cfdumper/main.c

```diff
@@ -1660,6 +1668,7 @@ static void printDisassembledMethod(J9CfrClassFile* classfile, J9CfrMethod* meth
 				case CFR_BC_putstatic:
 				case CFR_BC_getfield:
 				case CFR_BC_putfield:
+				case CFR_BC_withfield:
```

Comment:
these changes should be ifdef'd. If the JVM is not VT enabled, the withfield, defaultValue bytecodes should be treated as unimplemented bytecodes